Library/ViewModels/RewardsCollectionViewModel.swift

```diff
@@ -228,6 +228,40 @@ public final class RewardsCollectionViewModel: RewardsCollectionViewModelType,
           refTag: refTag
         )
       }
+
+    // FB CAPI
+    let graphUser = self.viewDidAppearProperty.signal.ignoreValues()
```

Comment:
Similar to above, let's remove this heavy call here and add it back in later with `AppEnvironment.current.loggedInUserEmail`.